MathlibExtras/DotNotation.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2023 Martin Dvorak. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Martin Dvorak, Kyle Miller
+-/
+import Mathlib.Algebra.BigOperators.Multiset.Basic
+
+@[app_unexpander List.take]
+def List.take.unexpander : Lean.PrettyPrinter.Unexpander
+  | `($_ $f $μ) => `($(μ).$(Lean.mkIdent `take) $f)
```

Comment:
Could you go through and make sure the names reflect what they're for? It would make it easier to evaluate whether these unexpanders are correct.

```suggestion
  | `($_ $n $xs) => `($(xs).$(Lean.mkIdent `take) $n)
```